packages/react-serialize/CHANGELOG.md

```diff
@@ -4,10 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-<!-- Unreleased changes should go to UNRELEASED.md -->
+## Unreleased
```

Comment:
actually, can we just deprecate `react-serialize` all together?

You can try out the new deprecation guide
https://github.com/Shopify/quilt/blob/master/documentation/guides/package-deprecation.md